tests/e2e-pw/specs/shopper/shopper-myaccount-saved-cards.spec.ts

```diff
@@ -21,11 +21,43 @@ import {
 } from '../../utils/shopper';
 import RestAPI from '../../utils/rest-api';
 
+type TestVariablesType = {
+	[ key: string ]: {
+		card: typeof config.cards.basic;
+		address: {
+			country: string;
+			postalCode: string;
+		};
+		products: [ string, number ][];
+	};
+};
+
 const cards = {
-	basic: config.cards.basic,
-	'3ds': config.cards[ '3ds' ],
-	'3ds2': config.cards[ '3ds2' ],
-} as { [ key: string ]: typeof config.cards.basic };
+	basic: {
+		card: config.cards.basic,
+		address: {
+			country: 'US',
+			postalCode: '94110',
+		},
```

Comment:
If the address is always the same we can omit it here, right?

Reply:
Yep, we can omit them, I thought about removing them too, but left it intact because I think we can leave them ready for change if we need to change, for example we might want to add something that tests with a different country and post code etc. 

And as the real reason, I don't want to trigger another E2E run and wait for 40 minutes, I prefer them staying :D